### PR TITLE
make auto resize a bit smarter. fixup styling in update / pause dialogs

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: desktop/renderer/remote-component-helper.js [hunk 1]
```diff
@@ -4,20 +4,42 @@ import {globalStyles} from '../shared/styles/style-guide'
...
+          browserWindow.setContentSize(browserWindow.getSize()[0], element.scrollHeight + 2 * element.offsetTop + 2 * globalStyles.windowBorder.borderWidth)
+          browserWindow.setResizable(originalResizableState)
+        })
       }
     } catch (e) {
       console.error('error in resizing frame', e)
     }
+
+    function getStableHeight (element, left, delay, done) {
+      const height = element.scrollHeight
```
Comment (chrisnojima) [inline desktop/renderer/remote-component-helper.js]: kinda janky but it works. we wait and measure and resize when things settle down

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: desktop/renderer/remote-component-helper.js (hunk 1 above)
Comment (MarcoPolo) [inline desktop/renderer/remote-component-helper.js]: This is fine, but is it possible to listen to some window event to know when things have settled?

Comment (chrisnojima) [inline desktop/renderer/remote-component-helper.js]: there isn't that i know of

Comment (chrisnojima) [inline desktop/renderer/remote-component-helper.js]: i tried listening for dom ready and etc but i think it fires before the react stuff is really done rendering (like before we have props maybe)

**Comment by MarcoPolo** [comment]
:+1:

**Pull request merged**